src/DssSpell.sol

```diff
@@ -38,94 +119,252 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
     // uint256 internal constant X_PCT_RATE      = ;
-    uint256 internal constant RAY       = 10 ** 27;
 
-    address internal immutable MCD_SPOT = DssExecLib.spotter();
+    uint256 internal constant WAD                            = 10 ** 18;
+    uint256 internal constant MILLION                        = 10 ** 6;
+    uint256 internal constant DEBT_CEILING_UNITS             = 10 ** 2;
```

Comment:
Confirmed that this is a legitimate multiplier (100) and not malicious.

[Refer to here](https://github.com/aave/aave-v3-core/blob/29ff9b9f89af7cd8255231bc5faf26c3ce0fb7ce/contracts/protocol/libraries/configuration/ReserveConfiguration.sol#L65) for how Reserve Config uses 10 ** 2 for `DEBT_CEILING_DECIMALS`.

I was suspicious for a second because `DEBT_CEILING_UNITS` had never been used elsewhere on GitHub but it turns out this is a naming thing! :)

Reply:
Feel free to rename. Idc what it's called.